### ISSUE TITLE
Sluggish behavior and freezes of Sanboxie-Plus GUI?

### ISSUE THREAD

**Issue by ViperCode7** [sandboxie-plus/Sandboxie#4035]
### Describe what you noticed and did

After a while, the GUI begins to freeze up. It usually doesn't happen on reboots/cold boots, but after a while, since I let my laptop run for days, Sandboxie starts to behave like that. This also involves the certificate popup, which was the 1st time I had witnessed some sense of error.

Troubleshooting procedure 'SBIEMSG'
Further information:
Sandboxie-Plus Version: 1.13.7
Operating System Version: 10.0.19045-x86_64
MsgCode: 2203
MsgData: *MANPROXY_00000001 - (null) [00000102],
ProcessName: PID 264

Well, even after trying to report the issue, the troubleshooting wizard started to freeze up for like 20 seconds.
The main GUI does the same.
And as I've said prior, the certificate nag screen also did this, for the 1st time now.

I was using it to start multiple upload sessions with a cloud software which otherwise wants to run standalone, by using Sandboxie, it can run as multiple instances. I usually use 4 sandboxes to run it so that it mostly maxes out my upload speeds.

However, these issues were showing up regardless of apps or how many sandboxes were used, most of the time I try game installers from repacking groups and GOG installers, so one at a time, max 2 sandboxes. While the apps run fine, after they're done I usually remove the contents for the box and this is where most of the freezing up is experienced, before, after, in-between when nothing is running in Sandboxie.

### How often did you encounter it so far?

Daily, after some time of use.

### Expected behavior

Not freeze up.

### Affected program

Any

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

1.13.7

### Is it a new installation of Sandboxie?

I have been using the same version for some time.

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

10 22H2 x64

### In which Windows account you have this problem?

A local account (Standard user)., A local account (Administrator)., An account with secure desktop turned off for UAC prompts.

### Please mention any installed security software

Eset IS

**Comment by ViperCode7** [comment]
Recent crash after trying to disable three common messages, confirmation window was freezing after each press, eventually the GUI crashed.

> Faulting application name: SandMan.exe, version: 1.13.7.0, time stamp: 0x662f4490
Faulting module name: Qt5Core.dll, version: 5.15.13.0, time stamp: 0x661b7dbc
Exception code: 0xc0000005
Fault offset: 0x000000000000bd73
Faulting process id: 0x4020
Faulting application start time: 0x01dac983c08425bd
Faulting application path: C:\Program Files\Sandboxie\SandMan.exe
Faulting module path: C:\Program Files\Sandboxie\Qt5Core.dll
Report Id: 5ed765ef-dcdb-4b24-85fa-50f8f6f21757
Faulting package full name: 
Faulting package-relative application ID:

**Comment by offhub** [comment]
Could you please provide a crash dump?
> %LOCALAPPDATA%\Sandboxie-Plus

**Comment by ViperCode7** [comment]
[SandMan.exe.16416.dmp](https://github.com/user-attachments/files/16042067/SandMan.exe.16416.dmp)

**Comment by ViperCode7** [comment]
[SandMan-v1.13.7 29.06.2024 22-03-57,763.dmp](https://github.com/user-attachments/files/16052306/SandMan-v1.13.7.29.06.2024.22-03-57.763.dmp)
Sorry, this is the one form that location requested... the previous one was from CrashDumps

**Comment by ViperCode7** [comment]
Yes, this is still present in 1.14.10. Every time I open the UI it freezes...

**Comment by offhub** [comment]
1. Open File Explorer and enter the following path in the address bar:
  ```
  %LocalAppData%\Sandboxie-Plus
  ```

2. Locate and open `Sandboxie-Plus.ini` using a text editor, such as Notepad.

3. Look for the line:
  ```
  AutoRunSoftCompat=true
  ```

4. Replace the line with:
  ```
  AutoRunSoftCompat=false
  ```
  > **Note**: If the line doesn't exist, insert it under the `[Options]` section.

5. Save the file.

6. Launch the Sandboxie-Plus user interface and see if it resolves the freezing issue.

**Comment by githubusername2024** [comment]
The suggested solution fixed the problem for me but I have no idea how to run software compatibility on demand

**Comment by offhub** [comment]
If you run it manually, you'll still encounter the same issue, so it's essential for us to identify the cause of the freezing. I’ll provide some test `Templates.ini` files for this later.

**Comment by ViperCode7** [comment]
I've run so far all templates, the UI freezes on start (when it goes from NOT connected to running). Other than that I don't see any issues when it doesn't run anything.

When I've said that it freezes I didn't meant after a clean start, like point 4. Restart SandMan above.
It freezes for instance when I run some apps, get the UAC prompt, either Cancel or allow but then close the app and the interface freezes.

**Comment by offhub** [comment]
There may be freezes during box operations, but these should not bother you. Try after enabling the setting below.

> Sandboxie-Plus > Options > Global Settings > Gneral Config > Run box operations asynchronously whenever possible (like content deletion)
